redis_broker.go

```diff
@@ -282,8 +325,14 @@ func (t *RedisBroker) enqueue(ctx context.Context, msg *Message, dynamicOn bool)
 	}
 
```

Comment:
It is recommended to use `TxPipelined` to ensure data consistency. In the scenario where `filter.Add` succeeds but `client.XAdd` fails, using the same ID in the next request will result in a `duplicate ID` error, which is not what we expect because the previous transaction with this ID did not actually start executing.